spec/system/admin/enterprises_spec.rb

```diff
@@ -874,9 +874,18 @@
         click_button "Change Package"
 
         page.find('a', class: 'selected', text: "PRODUCER PROFILE")
-        expect(enterprise.is_primary_producer).to eq true
+
+        # a primary producer:
+        expect(enterprise.reload.is_primary_producer).to eq true
+
+        # which is not selling:
         expect(enterprise.reload.sells).to eq('none')
 
+        # then, this should imply
+        # producer_profile_only sto be true
```

Comment:
Typo

Reply:
Ohh :see_no_evil: 
Changed and squashed it - thank you @rioug